app/src/pages/explorer/parameterized-query/templates/PQ04.ttl

```diff
@@ -0,0 +1,73 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix sp: <http://spinrdf.org/sp> .
+@prefix spin: <http://spinrdf.org/spin#> .
+@prefix spl: <http://spinrdf.org/spin#> .
+@prefix whyis: <http://vocab.rpi.edu/whyis/> .
+@prefix materialsmine_templates: <http://materialsmine.org/query/> .
+
+materialsmine_templates:PQ04 a whyis:SparqlTemplate;
+  spin:labelTemplate "Show me <b>research articles</b> containing the word {?Text} in the article title.";
+  sp:text """
+PREFIX dct: <http://purl.org/dc/terms/>
+SELECT DISTINCT * WHERE {
+  ?doi a dct:BibliographicResource ;
+       dct:created ?Year ;
+       dct:title ?Title
+  FILTER (REGEX(LCASE(STR(?Title)), ?Text))
```

Comment:
Is there any way to ignore casing for this? I noticed that it isn't returning any results even though we definitely have articles that technically fit those queries
e.g, https://onlinelibrary.wiley.com/doi/10.1002/adma.200801758 has "Dielectric" in the title but selecting "dielectric" in this template doesn't return anything

Reply:
this is an interesting bug. Replacing `?Text` with `"dielectric"` in the filter clause makes it yield results. This seems to be the case in other queries with FILTER params. Will investigate further

Reply:
fixed